javascript/building-blocks/tasks/events/events1-download.html

```diff
@@ -30,6 +30,13 @@
 
   <script>
     const btn = document.querySelector('.off');
+    const newText = btn ? "Machine is off" : "Machine is on"
+    const clickButton = function (newText) => {
+      clickButton.addEventListener("onclick", (newText) => {
+        console.log (newText)
+        
+      })
+    })
 
```

Comment:
You also have to call the clickButton variable with function with the newText as its argument to make sure it is working properly. Something like this:

clickButton(newText);